tests/trainer/test_trainer.py

```diff
@@ -1522,12 +1549,17 @@ def on_predict_epoch_end(self, trainer, pl_module, outputs):
         super().on_predict_epoch_end(trainer, pl_module, outputs)
 
 
-def predict(tmpdir, accelerator, gpus, num_processes, model=None, plugins=None, datamodule=True, pbrr=None):
+def predict(
+    tmpdir, accelerator, gpus, num_processes, model=None, plugins=None, datamodule=True, pbrr=None, use_callbacks=True
+):
```

Comment:
need to be careful that we don't overload these test helper functions with complexity. If the test functions are too complex we would need tests for the tests and so this goes in circles xD

Reply:
also rather keep them protected so noone would import them...